### PR TITLE
fix several typos

### PR DESCRIPTION
**Pull request by stevenzwu**

### CODE REVIEW

File: src/main/java/rx/observables/AbstractOnSubscribe.java [hunk 1]
```diff
@@ -98,7 +98,7 @@
  * <pre><code>
  * Iterable<T> iterable = ...;
  * AbstractOnSubscribe.create(s -> {
- *   Iterator<T> it = s.state();
+ *   Iterator<T> it = iterable.iterator();
```
Comment (akarnokd) [inline src/main/java/rx/observables/AbstractOnSubscribe.java]: There is no typo here, the per-subscriber state of iterator is created on line 108.

**Comment by benjchristensen** [comment]
Can you please rebase these 3 commits into 1?

**Comment by stevenzwu** [comment]
first time does rebase/squash. seems to get it correct with one commit now

**Comment by akarnokd** [comment]
Thanks.

**Pull request merged**